core/pki/document_test.go

```diff
@@ -181,4 +181,13 @@ func TestDocument(t *testing.T) {
 		require.NoError(err)
 		require.True(bytes.Equal(d, d2))
 	}
+
+	// test that we can unmarshal a document stripped of signatures
```

Comment:
we could maybe also take the opportunity to test that `StripSignatures()` lives up to its name here, and additionally that the **verifying** deserializer can no longer read it.

Reply:
no. those changes were made a long time ago. we can mess with it in another PR.